src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -1288,6 +1298,28 @@ private bool IsQueryMatch(string query, string symbolName)
 
         #region Event Handlers
 
+        private FoldingRange[] Fold(
+            string documentUri)
+        {
+            // TODO Should I be using dynamic registrations?
+            if (this.currentSettings.CodeFolding.Enable != true) { return null; }
+            // TODO Surely there's a better way to do type adapters?
+            List<FoldingRange> result = new List<FoldingRange>();
+            foreach (FoldingReference fold in TokenOperations.FoldableRegions(
+                editorSession.Workspace.GetFile(documentUri).ScriptTokens,
+                this.currentSettings.CodeFolding.ShowLastLine))
+            {
+                FoldingRange item = new FoldingRange();
```

Comment:
Ditto @rjmholt comment above about using `var`. The rule we are all trying to follow (now) in this codebase is that if the type is obvious from the RHS of the assignment, then use `var` e.g.:
```
var str = "Some string literal is obviously a string";
var num = 42; // Obviously an int
var list = new List<string>(); // Obviously a List of type string
```

Reply:
Fixed